modules/structs/attachment.go

```diff
@@ -3,6 +3,7 @@
 // license that can be found in the LICENSE file.
 
 package structs // import "code.gitea.io/gitea/modules/structs"
+
```

Comment:
why this change?

Reply:
between `package` and `import` it should have a blank line but not.